### ISSUE TITLE
Add a dark mode to the editor

### ISSUE THREAD

**Issue by Nightcaat** [mroyale/mroyale-client#11]
Yeah

**Comment by terminalarch** [comment]
Yeah so no progress on this as I've been working on mroyale/mroyale-server-new. Lmk if this and the other issue should be closed

**Comment by Nightcaat** [comment]
This will be added in Deluxe 1.0.0

**Comment by Nightcaat** [comment]
Reopening, this will be backported from Deluxe in a future commit
